www/index.html

```diff
@@ -466,6 +466,8 @@ <h3>Display settings</h3>
                                         <b>Select app theme</b> (content inversion is experimental):
                                     </label>
                                     <select class="form-control" id="appThemeSelect">
+                                        <option class="auto" value="auto_invert">Auto (generic)</option>
```

Comment:
I find disturbing that the `auto*` options do not match the `dark*` options.

There is no `auto` option that allows to switch to `Dark (app only)` when the OS is in dark mode. I know we did not initially ask for it, but shouldn't it be added?

And the labels should match better. I suggest for example: `Auto (generic invert)` and `Dark (generic invert)`, `Auto (app only)` and `Dark (app only)`, `Auto (wikimedia invert)*` and `Dark (wikimedia invert)*`

Reply:
This ought to be pretty easy to implement within the existing schema.